src/app/views/dashboard-view/views/settings-view/settings-general-tab/settings-general-tab.component.html

```diff
@@ -74,6 +74,46 @@ <h2 translate>settings.general.adminPrivileges.title</h2>
     </div>
   </div>
 </div>
+
+<div class="setting-category">
+  <h2 translate>settings.general.systemTray.title</h2>
+  <div class="settings">
+    <div class="setting-row">
+      <div class="setting-row-label" translate>
```

Comment:
```suggestion
      <div class="setting-row-label">
```
Nit: unneeded translate directive